### ISSUE TITLE
Jdk 9/10 fails to build

### ISSUE THREAD

**Issue by proubatsis** [OpenLiberty/guide-maven-multimodules#21]
This is the error from `mvn clean install` on travis:
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] Source option 5 is no longer supported. Use 6 or later.
[ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] io.openliberty.guides.multimodules.modules ......... SUCCESS [  2.664 s]
[INFO] io.openliberty.guides.multimodules.jar ............. FAILURE [  5.082 s]
[INFO] io.openliberty.guides.multimodules.war ............. SKIPPED
[INFO] io.openliberty.guides.multimodules.ear ............. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7.944 s
[INFO] Finished at: 2018-06-19T19:17:55Z
[INFO] Final Memory: 13M/44M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project io.openliberty.guides.multimodules.jar: Compilation failure: Compilation failure: 
[ERROR] Source option 5 is no longer supported. Use 6 or later.
[ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :io.openliberty.guides.multimodules.jar
The command "mvn clean install" exited with 1.
```

https://travis-ci.org/OpenLiberty/guide-maven-multimodules/jobs/394252816

**Comment by proubatsis** [comment]
**Tested with:**

* Open JDK 9
* Oracle JDK 9
* Open JDK 10
* Oracle JDK 10

**Comment by proubatsis** [comment]
Fixed in #23

**Comment by proubatsis** [comment]
**Build:** 18.0.0.1/wlp-1.0.20.cl180120180309-2209
